flintrock/templates/spark/conf/spark-defaults.conf

```diff
@@ -0,0 +1 @@
+spark.jars.packages    org.apache.hadoop:hadoop-aws:{hadoop_version}
```

Comment:
This line here will ensure that Spark automatically loads the necessary package to enable S3 access.